File: typescript/5kyu/Human_Readable_Time.ts
export const humanReadable = (seconds: number): string => {
  const totMins = Math.floor(seconds / 60);
  const checkDigitLength = (num: number) =>
    String(num).length === 1 ? `0${num}` : num;

  const secs = checkDigitLength(seconds % 60);
  const hrs = checkDigitLength(Math.floor(totMins / 60));
  const mins = checkDigitLength(totMins % 60);

  return `${hrs}:${mins}:${secs}`;
};

File: python/5kyu/Human_Readable_Time.py
import math

def make_readable(seconds):
    total_minutes = math.floor(seconds / 60)
    def check_digit_len(num):
        if len(str(num)) == 1:
            return "0" + str(num)
        else:
            return str(num)

    secs = check_digit_len(seconds % 60)
    hrs = check_digit_len(math.floor(total_minutes / 60))
    mins = check_digit_len(total_minutes % 60)

    return hrs+":"+ mins+":"+secs
